docs_source_files/content/introduction/on_test_automation.en.md

```diff
@@ -3,9 +3,9 @@ title: "On test automation"
 weight: 2
 ---
 
-First, start by asking yourself whether or not you really need to use a browser.
```

Comment:
Same comments as in the DE version file.

Reply:
Reverted for other pages having these changes.